modules/oe_content_sub_entity/modules/oe_content_sub_entity_person/tests/src/Kernel/PersonReferenceFormatterTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_sub_entity_person\Kernel;
+
+use Drupal\KernelTests\Core\Entity\EntityKernelTestBase;
+use Drupal\node\NodeInterface;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Drupal\Tests\node\Traits\NodeCreationTrait;
+use Drupal\Tests\oe_content\Traits\EntityReferenceRevisionTrait;
+use Drupal\Tests\sparql_entity_storage\Traits\SparqlConnectionTrait;
+
+/**
+ * Tests Persons reference formatter.
+ */
+class PersonReferenceFormatterTest extends EntityKernelTestBase {
+
+  use NodeCreationTrait;
+  use EntityReferenceRevisionTrait;
+  use ContentTypeCreationTrait;
+  use SparqlConnectionTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'node',
+    'link',
+    'entity_reference_revisions',
+    'sparql_entity_storage',
+    'rdf_skos',
+    'composite_reference',
+    'media',
+    'image',
+    'text',
+    'options',
+    'datetime',
+    'typed_link',
+    'field_group',
+    'maxlength',
+    'description_list_field',
+    'inline_entity_form',
+    'oe_content_entity_contact',
+    'oe_content_social_media_links_field',
+    'oe_content',
+    'oe_content_departments_field',
+    'oe_content_person',
+    'oe_content_timeline_field',
+    'oe_content_organisation',
+    'oe_content_sub_entity',
+    'oe_content_sub_entity_document_reference',
+    'oe_content_sub_entity_person',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->setUpCurrentUser(['uid' => 1]);
+    $this->setUpSparql();
+
+    $this->installEntitySchema('oe_person');
+    $this->installEntitySchema('oe_document_reference');
```

Comment:
Why? Do we need this? Please ensure it's not just all copy pasted from the authors tests